test/github-release.ts

```diff
@@ -451,72 +450,161 @@ describe('GitHubRelease', () => {
         );
       }
       expect(failed).to.be.true;
-      requests.done();
     });
-  });
 
-  describe('extractLatestReleaseNotes', () => {
-    it('handles CHANGELOG with old and new format entries', () => {
-      const changelogContent = readFileSync(
-        resolve(fixturesPath, './CHANGELOG-old-new.md'),
-        'utf8'
-      ).replace(/\r\n/g, '\n');
-      const latestReleaseNotes = GitHubRelease.extractLatestReleaseNotes(
-        changelogContent,
-        'v1.0.0'
-      );
-      snapshot(latestReleaseNotes);
-    });
+    it('parses version from PR title (detectReleaseVersionFromTitle impl: base)', async () => {
```

Comment:
This seems like a nice simplification over some other approaches we've discussed, e.g., loading the version form `package.json` or `setup.py`.